### ISSUE TITLE
Keyboard navigation to Today and Cancel

### ISSUE THREAD

**Issue by manolo** [vaadin/vaadin-date-picker#195]
There is no way to focus today and cancel buttons using keyboard. Probably tab should be the appropriate key.

**Comment by jouni** [comment]
Cancel is not that crucial, since you can do the same action using the ESC key. But it should be possible to use the Today shortcut using keyboard only, and if we make the button Today button focusable, then it makes sense to make the cancel button focusable as well. And eventually we might have the explicit "OK" button as well, which should be focusable also.

**Comment by jouni** [comment]
@platosha, will this be covered by the work you are doing in #70 ?

**Comment by jouni** [comment]
This issue was fixed by #247
